docs/tutorials/gluon/naming.md

```diff
@@ -0,0 +1,236 @@
+```
+
+    sequential1_ sequential1_alexnet0_ sequential1_alexnet1_
+
+
+## Saving and loading
+
+Because model0 and model1 have different prefixes, their Parameters also have different names:
+
+
+```python
+print(model0.collect_params(), '\n')
+print(model1.collect_params())
+```
+
+    model0_ (
+      Parameter model0_dense0_weight (shape=(20L, 20L), dtype=<type 'numpy.float32'>)
+      Parameter model0_dense0_bias (shape=(20L,), dtype=<type 'numpy.float32'>)
+      Parameter model0_dense1_weight (shape=(20L, 20L), dtype=<type 'numpy.float32'>)
+      Parameter model0_dense1_bias (shape=(20L,), dtype=<type 'numpy.float32'>)
+      Parameter model0_mydense_weight (shape=(20L, 20L), dtype=<type 'numpy.float32'>)
+      Parameter model0_mydense_bias (shape=(20L,), dtype=<type 'numpy.float32'>)
+    ) 
+    
+    model1_ (
+      Parameter model1_dense0_weight (shape=(20, 0), dtype=<type 'numpy.float32'>)
+      Parameter model1_dense0_bias (shape=(20,), dtype=<type 'numpy.float32'>)
+      Parameter model1_dense1_weight (shape=(20, 0), dtype=<type 'numpy.float32'>)
+      Parameter model1_dense1_bias (shape=(20,), dtype=<type 'numpy.float32'>)
+      Parameter model1_mydense_weight (shape=(20, 0), dtype=<type 'numpy.float32'>)
+      Parameter model1_mydense_bias (shape=(20,), dtype=<type 'numpy.float32'>)
+    )
+
+
+As a result if you try to save parameters from model0 and load it with model1, you'll get an error due to unmatching names:
+
+
+```python
+model0.collect_params().save('model.params')
+try:
+    model1.collect_params().load('model.params', mx.cpu())
+except Exception, e:
+    print(e)
+```
+
+    Parameter 'model1_dense0_weight' is missing in file 'model.params', which contains parameters: 'model0_mydense_weight', 'model0_dense1_bias', 'model0_dense1_weight', 'model0_dense0_weight', 'model0_dense0_bias', 'model0_mydense_bias'. Please make sure source and target networks have the same prefix.
+
+
+To solve this problem, we use `save_params`/`load_params` instead of `collect_params` and `save`/`load`. This way, the unmatching part of parameter names (`model0_` and `model1_`) are stripped and only the matching part is saved.
+
+
+```python
+model0.save_params('model.params')
+model1.load_params('model.params', mx.cpu())
+print(mx.nd.load('model.params').keys())
+```
+
+    ['dense0_bias', 'mydense_weight', 'mydense_bias', 'dense1_bias', 'dense1_weight', 'dense0_weight']
+
+
+**Nevertheless, you are still recommended to manually specify the prefix of the top level Block (i.e. `model = Model(prefix='mymodel_')`) to avoid any potential problem during saving and loading.**
+
+## Replacing Blocks from networks and fine-tuning
+
+Sometimes you may want to load a pretrained model, and replace certain Blocks in it for fine-tuning.
+
+For example, the alexnet in model zoo has 1000 output dimensions, but maybe you only have 100 classes in your application.
+
+To see how to do this, we first load an pretrained alexnet.
```

Comment:
a pretrained*

Reply:
maybe `alexnet` if you want to refer to it now as an object instead of a model from the zoo, where it would be AlexNet.